tests/contents/server.mjs

```diff
@@ -17,7 +17,7 @@ const routeObj = urls.reduce((acc, elt) => {
  * @param {number} port
  * @returns {Object}
  */
-export default function createContentServer(port) {
+export function createContentServer(port) {
```

Comment:
Why not `export function createContentServer(port = DEFAULT_CONTENT_SERVER_PORT) {`.

This way an importer could still overwrite the port if it wants to, just it's not mandatory anymore